aiida/cmdline/commands/computer.py

```diff
@@ -228,6 +329,254 @@ def disable_computer(ctx, only_for_user, computer):
+                    continue
+
+            if computer.is_enabled():
+                if is_configured:
+                    configured_str = ""
+                    if is_user_enabled:
+                        symbol = "*"
+                        color = 'green'
+                        enabled_str = ""
+                        is_usable = True
+                    else:
+                        symbol = "x"
+                        color = 'red'
+                        enabled_str = "[DISABLED for this user]"
+                else:
+                    symbol = "x"
+                    color = 'reset'
+                    enabled_str = ""
+                    configured_str = " [unconfigured]"
+            else:  # GLOBALLY DISABLED
+                symbol = "x"
+                color = 'red'
+                if is_configured and not is_user_enabled:
+                    enabled_str = " [DISABLED globally AND for this user]"
+                else:
+                    enabled_str = " [DISABLED globally]"
+                if is_configured:
+                    configured_str = ""
+                else:
+                    configured_str = " [unconfigured]"
+
+            if parsable:
+                echo.echo(click.style("{}".format(name), fg=color))
+            else:
+                if (not only_usable) or is_usable:
+                    echo.echo(click.style("{} ".format(symbol), fg=color), nl=False)
+                    echo.echo(click.style("{} ".format(name), bold=True, fg=color), nl=False)
+                    echo.echo(click.style("{}{}".format(enabled_str, configured_str), fg=color))
+
+    else:
+        echo.echo("# No computers configured yet. Use 'verdi computer setup'")
+
+
+@verdi_computer.command('show')
+@arguments.COMPUTER()
+@with_dbenv()
+def computer_show(computer):
+    """
+    Show information on a given computer
+    """
+    return echo.echo(computer.full_text_info)
+
+
+@verdi_computer.command('rename')
+@arguments.COMPUTER()
+@arguments.LABEL('NEW_NAME')
+@with_dbenv()
+def computer_rename(computer, new_name):
+    """
+    Rename a computer
+    """
+    from aiida.common.exceptions import UniquenessError, ValidationError
+
+    old_name = computer.get_name()
+
+    if old_name == new_name:
+        echo.echo_critical("The old and new names are the same.")
+
+    try:
+        computer.set_name(new_name)
+        computer.store()
+    except ValidationError as error:
+        echo.echo_critical("Invalid input! {}".format(error.message))
+    except UniquenessError as error:
+        echo.echo_critical("Uniqueness error encountered! Probably a "
+                           "computer with name '{}' already exists"
+                           "".format(new_name))
+        echo.echo_critical("(Message was: {})".format(error.message))
+
+    echo.echo_success("Computer '{}' renamed to '{}'".format(old_name, new_name))
+
+
+@verdi_computer.command('test')
+@click.option(
```

Comment:
Use `aiida.cmdline.params.options.USER` option instead, overwriting only `required` and `help` kwargs.